### PR TITLE
Javadoc sample code generated

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java [hunk 1]
```diff
@@ -95,6 +95,16 @@ public DigitalTwinsServiceVersion getServiceVersion() {
     /**
      * Creates a digital twin.
      *
+     * <p><strong>Code Samples</strong></p>
```
Comment (barustum) [inline sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java:98]: Do you need that same header (Code Samples) for the constructor code snippet?

File: sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/snippets/CodeSnippetBase.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
+package com.azure.digitaltwins.core.snippets;
+
+public abstract class CodeSnippetBase {
+
+    //region DigitalTwinSnippets
+
```
Comment (barustum) [inline sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/snippets/CodeSnippetBase.java:6]: extra space here and before the region ends

**Review by barustum — APPROVED** [review]

**Comment by timtay-microsoft** [comment]
Can you add an Update-Snippets.ps1 like we have in .NET?

**Review by timtay-microsoft — APPROVED** [review]

**Comment by azabbasi** [comment]
> Can you add an Update-Snippets.ps1 like we have in .NET?

There is no need. Snippets are generated during build and are embedded in the javadocs

**Pull request merged**